src/config.ts

```diff
@@ -40,7 +46,28 @@ export const configConstants = {
     stage: "dev",
     prefix: "sls",
     localPort: 7071,
+    os: FunctionAppOS.WINDOWS,
+  },
+  dockerImages: {
+    node: {
+      "8": "DOCKER|microsoft/azure-functions-node8:2.0",
+      "10": "DOCKER|microsoft/azure-functions/node:2.0"
+    },
+    python: {
+      "3.6": "DOCKER|microsoft/azure-functions/python:2.0"
+    }
   },
+  defaultFuncIgnore: [
```

Comment:
Does this play nice with Node apps on linux?  If I don't use webpack I would need to ensure my `package.json` and `node_modules` are included in my deployment.

Reply:
I'll update this.